firefox-ios/firefox-ios-tests/Tests/XCUITests/MicrosurveyTests.swift

```diff
@@ -14,7 +14,9 @@ final class MicrosurveyTests: BaseTestCase {
     }
 
     func testShowMicrosurveyPromptFromHomepageTrigger() {
-        generateTriggerForMicrosurvey()
+        // Disabled due to issue in toggling between regular and private modes.
+        // https://github.com/mozilla-mobile/firefox-ios/issues/22295
+        // generateTriggerForMicrosurvey()
```

Comment:
@dragosb01 Could you please confirm that we don't have to `privateModeToggleButton` (or `AccessibilityIdentifiers.Browser.TopTabs.privateModeButton` for iPad) to trigger the microsurvey?

Reply:
Not sure about the microsurvey, these tests were not created by me. But if there is a fix in progress i say lets wait for it.

Reply:
These tests were created by Cydi, they are not still in Test Rail into the Full Functional, they live in a separate test suite https://mozilla.testrail.io/index.php?/suites/view/53811
@abodea  can these be added to the Full Funcional already? If so, please add smoke for those that correspond. If that happens @dragosb01 please see which ones match with these tests to mark them with the automation label. Thanks! 🙏

Reply:
@cyndichin has something changed with the Microsurveys tests?

Reply:
@isabelrios there is a bug with the toggle button and not specific to microsurvey system. The toggle button serves as a trigger for the survey. We don't have to use the `privateModeToggleButton` but it seemed to be the easiest way to visit the homepage at least twice. To trigger the survey, you can do other operations such as opening a new tab or navigating elsewhere and then coming back to home.

However, I think that it's great that this UI test caught the bug [here](https://mozilla-hub.atlassian.net/browse/FXIOS-10155) and failed. I don't think we have any other UI tests that would've caught it right? So in the case, maybe I would recommend leaving in it? What do you think?

Reply:
Thanks @cyndichin ! I understand this now! I would say we have more tests that tap on that button, let me check. 
Since these tests do not run as part of Smoketest I am fine with leaving them as they are until there is a fix, looks like is a important issue and the fix will land soon 🤞

Reply:
thanks @isabelrios, @OrlaM has a fix!! https://github.com/mozilla-mobile/firefox-ios/pull/22341